src/extensionWeb.ts

```diff
@@ -4,17 +4,17 @@
  */
 
 import * as vscode from 'vscode'
-
-// The following is required so that the copyFiles script does not fail.
-// I'm assuming this generates something when run that the script can use.
-import * as nls from 'vscode-nls'
-nls.loadMessageBundle()
+import { setInBrowser } from './browser/utils'
 
 export async function activate(context: vscode.ExtensionContext) {
-    vscode.window.showInformationMessage(
-        'AWS Toolkit: Browser Mode Under Development. No features are currently provided',
-        { modal: true }
-    )
+    setInBrowser(true)
+    
+    // This causes the Extension Browser tests to fail.
+    // See the comment in src/testBrowser/testRunner.ts#stubVscodeWindow() for more info.
+    // vscode.window.showInformationMessage(
+    //     'AWS Toolkit: Browser Mode Under Development. No features are currently provided',
+    //     { modal: true }
```

Comment:
do the tests work if it's not modal?
```suggestion
    //     { modal: false }
```

Reply:
Oh nice, didn't realize this was the core of the issue. Fixed it